modules/test/playwright/pages/portal-language-override-web/LanguageOverridePage.ts

```diff
@@ -82,7 +92,7 @@ export class LanguageOverridePage {
 		}
 	}
 
-	async assertTranslationNotInListView({key}: TTranslation) {
+	async assertLanguageKeyNotInListView({key}: TLanguageKey) {
```

Comment:
Since this function only needs the key it's better not to pass the whole TLanguageKey object.